apps/prairielearn/src/ee/lib/lti13.ts

```diff
@@ -191,6 +196,45 @@ export type Lti13ClaimType = z.infer<typeof Lti13ClaimSchema>;
 
 export const STUDENT_ROLE = 'http://purl.imsglobal.org/vocab/lis/v2/membership#Learner';
 
+export async function getOpenidClientConfig(
+  lti13_instance: Lti13Instance,
+  options?: client.ModifyAssertionOptions,
+): Promise<client.Configuration> {
+  const keystore = lti13_instance.keystore as jose.JSONWebKeySet | null | undefined;
+  if (!keystore || !Array.isArray(keystore.keys) || keystore.keys.length === 0) {
+    throw new HttpStatusError(403, 'LTI 1.3 configuration error: no keys available in keystore');
```

Comment:
I'd think we want these to be 500s so that they bubble up to Sentry. We should never expect keys to be missing during normal operation. A plain `throw new Error(...)` is almost definitely sufficient.

Reply:
Addressed in fa49a1896f

Reply:
Assuming you meant 52b35a4, LGTM 👍